Tasks/CargoAuthenticateV0/cargoauthenticatemain.ts

```diff
@@ -92,7 +92,7 @@ async function main(): Promise<void> {
                     }      
                 }
                 // Default to internal registry if no token has been set yet
-                if (!currentRegistry) {
+                if (!currentRegistry && !tl.getVariable(tokenName)) {
```

Comment:
Consider adding a warning message or failing the task if the env variables have already been set

Reply:
I can add that, but since cargo auth iterates over all registries in the config file, all registry env variables will have some value after the first run. If the user just wants to authenticate one external registry at a time, though, we may not want to say that the registry env variable was already set, because it could be temporarily holding the internal auth value.

Reply:
That makes sense, lets at least have a warn or debug statement for easier

Reply:
I added a debug statement!